### PR TITLE
Launch RPC Gateway to 1% of Celo traffic

### PR DESCRIPTION
**Pull request by rollingtumbling**
Reworked https://github.com/Uniswap/routing-api/pull/518 to fix accidental provider swap issues.

We will launch to 1% of Celo traffic.

### CODE REVIEW

**Comment by rollingtumbling** [comment]
* **#523** <a href="https://app.graphite.dev/github/pr/Uniswap/routing-api/523?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rollingtumbling and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by jsy1218 — APPROVED** [review]

File: lib/config/rpcProviderProdConfig.json [hunk 1]
```diff
@@ -1 +1,8 @@
-[]
+[
+  {
+    "chainId": 42220,
+    "useMultiProviderProb": 0.01,
```
[the remark this answers is not in the document]
Comment (cgkol) [reply lib/config/rpcProviderProdConfig.json:4]: do we have good metrics to know what hte latency is for this 1% of the traffic?

[the remark this answers is not in the document]
Comment (cgkol) [reply lib/config/rpcProviderProdConfig.json:4]: and how it compares to rnormal traffic?

[the remark this answers is not in the document]
Comment (rollingtumbling) [reply lib/config/rpcProviderProdConfig.json:4]: I have RPC gateway dashboard to track the latency for the 1%. We can also just look at the RoutingAPI Dashboard about the overall latency for Celo.

Celo has 1% of the traffic of Base, and 0.5% of the traffic for Eth mainnet.

**Pull request merged**